spring-batch-core/src/main/java/org/springframework/batch/core/configuration/annotation/DefaultBatchConfigurer.java

```diff
@@ -93,12 +103,18 @@ public JobExplorer getJobExplorer() {
 	@PostConstruct
 	public void initialize() {
 		try {
-			if(dataSource == null) {
-				logger.warn("No datasource was provided...using a Map based JobRepository");
-
-				if(this.transactionManager == null) {
+			if(this.transactionManager == null) {
```

Comment:
This formatting seems inconsistent with the [Spring Framework Code Style](https://github.com/spring-projects/spring-framework/wiki/Code-Style). There should be a space between "if" and the paren. I mimicked what was already in the file because I felt it would be weird to mix the style within a single file. I'm happy to fix the formatting if you'd like me to, but I want to make sure the approach I took is acceptable before I go to that effort.